.github/workflows/node-gyp.yml

```diff
@@ -1,12 +1,11 @@
 name: node-gyp integration
 on:
   push:
-    branches: [ main ]
   pull_request:
-    branches: [ main ]
   workflow_dispatch:
+
 jobs:
-  integration:
+  node-gyp-integration:
```

Comment:
Renamed with prefix so that this can be cleared identified from merge requirements.